tsl/test/sql/compression_bgw.sql

```diff
@@ -74,30 +74,76 @@ CALL run_job(:compressjob_id);
 DROP TABLE IF EXISTS conditions CASCADE;
 
 --TEST 7
---compression policy for integer based partition hypertable
-CREATE TABLE test_table_int(time bigint, val int);
-SELECT create_hypertable('test_table_int', 'time', chunk_time_interval => 1);
-
-create or replace function dummy_now() returns BIGINT LANGUAGE SQL IMMUTABLE as  'SELECT 5::BIGINT';
-select set_integer_now_func('test_table_int', 'dummy_now');
-insert into test_table_int select generate_series(1,5), 10;
-alter table test_table_int set (timescaledb.compress);
-select add_compression_policy('test_table_int', 2::int) AS compressjob_id
-\gset
+--compression policy for smallint, integer or bigint based partition hypertable
```

Comment:
Could you add this for the distributed hypertable case as well?

Reply:
Added!